base_user_role/data/ir_module_category.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2016 ABF OSIELL <http://osiell.com>
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+
+<record model="ir.module.category" id="ir_module_category_role">
+    <field name='name'>User roles</field>
```

Comment:
I don' think this is used anywhere, and I think that we should NOT create this module category as it seems like other similar modules in Odoo core are using the category Tools or Extra Tools.

Reply:
This module category aims to be linked to role/group you create in your own custom module (https://github.com/odoo/odoo/blob/13.0/odoo/addons/base/models/res_users.py#L101) if you want. By doing this you'll have the section 'User roles' in the "Access rights" tab on the user form which will regroup all groups related to role. Without it, all these groups will be in the 'Other' section.
I could add a comment above this XML record about that.